tasks/eva_2230/check_RS_release_JSON_assumptions.py

```diff
@@ -0,0 +1,140 @@
+import bz2
+import click
+import json
+import traceback
+
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.logger import logging_config
+from pymongo import MongoClient
+
+
+logging_config.add_stdout_handler()
+logger = logging_config.get_logger(__name__)
+
+
+updated_rs_that_should_not_exist_in_EVA = []
+non_updated_rs_that_should_exist_in_EVA = []
+min_rs_records_to_check = 1000
+
+
+def get_mongo_query_result(rs_to_find, mongo_connection_handle):
+    results = []
+    dbsnpCVEHandle = mongo_connection_handle["eva_accession_human_sharded"]["dbsnpClusteredVariantEntity"]
+    dbsnpCVOEHandle = mongo_connection_handle["eva_accession_human_sharded"]["dbsnpClusteredVariantOperationEntity"]
```

Comment:
Why do you need to check both the `ClusteredVariantEntity` and the `ClusteredVariantOperationEntity`?

Reply:
There are some RSs that go to the CVOE collection because of locus collision. See RS 1475295477.

![image](https://user-images.githubusercontent.com/25152678/99257774-72c3ff00-280f-11eb-9b93-c7ade709a072.png)